src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    font-weight: $font-weight-400;
```

Comment:
font-weight: 400 - it's initial value - check it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/9bf5a0be-c272-4aa3-a500-06a06849809a)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/cd8ca86e-4f92-4137-b3bc-676133544f54)